snf-ui/Brocfile.js

```diff
@@ -51,10 +52,22 @@ app.import('bower_components/quill/quill.js');
 app.import('bower_components/quill/quill.core.css');
```

Comment:
Quill is removed from bower.json. Remaining imports result in failed builds as quill files cannot be located by ember-cli